src/CommunityToolkit.Aspire.Client.MassTransit.RabbitMQ/CommunityToolkit.Aspire.Client.MassTransit.RabbitMQ.csproj

```diff
@@ -0,0 +1,14 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+    <PropertyGroup>
+      <AdditionalPackageTags>Client Aspire MassTransit RabbitMQ</AdditionalPackageTags>
+      <Description>A .NET Aspire client integration for MassTransit RabbitMQ.</Description>
+      <ImplicitUsings>enable</ImplicitUsings>
+      <Nullable>enable</Nullable>
+    </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="MassTransit.RabbitMQ" Version="8.3.2" />
```

Comment:
Please ensure packages are added using NuGet tooling and use the version defined in `Directory.Package.props`